packages/amplify-graphql-searchable-transformer/streaming-lambda/python_streaming_function.py

```diff
@@ -14,19 +14,19 @@
 from boto3.dynamodb.types import TypeDeserializer
```

Comment:
There probably needs to be a step to add these env vars before this is released.

Reply:
I think `packages/amplify-graphql-searchable-transformer/src/cdk/create-streaming-lambda.ts` will need to be updated as well.